CHANGELOG-3.0.md

```diff
@@ -0,0 +1,119 @@
+# Release notes for master
+
+[Documentation](https://docs.k8s.io/docs/home)
+
+# Changelog since v2.2.0
+
+## Urgent Upgrade Notes 
+
+### (No, really, you MUST read this before you upgrade)
+
+- Update volumeAttachment to v1.
+- RBAC policy was updated to allow the external-attacher to patch VolumeAttachment.Status ([#200](https://github.com/kubernetes-csi/external-attacher/pull/200), [@cwdsuzhou](https://github.com/cwdsuzhou))
+
+## Changes by Kind
+
+### Feature
+
+- Use GA version of CSINode object. ([#193](https://github.com/kubernetes-csi/external-attacher/pull/193), [@bertinatto](https://github.com/bertinatto))
+
+### Bug or Regression
+
+- Fixed update of VolumeAttachment finalizers ([#227](https://github.com/kubernetes-csi/external-attacher/pull/227), [@mattcary](https://github.com/mattcary))
```

Comment:
This bug was introduced by changing to v1.VolumeAttachment right? So this bug didn't exist in a previously released version?

Reply:
Removed